pulp_container/app/cache.py

```diff
@@ -72,3 +73,19 @@ def find_base_path_cached(request, cached):
         except ObjectDoesNotExist:
             raise RepositoryNotFound(name=path)
         return distro.base_path
+
+
+class FlatpakIndexStaticCache(SyncContentCache):
+    def __init__(self, expires_ttl=None, auth=None):
+        updated_keys = (QUERY_KEY,)
+        super().__init__(
+            base_key="/index/static", expires_ttl=expires_ttl, keys=updated_keys, auth=auth
```

Comment:
`EXPIRES_TTL` is set to 600 seconds by default if we pass there `None`. Is this expected?

Reply:
I have no idea what a good value would be, and naively assumed the default would fit.  Any suggestions?

Reply:
I would make it last even for 1 or 6 hours. It is about your preferences.

Reply:
I think I'm completely incompetent to give any preferences here :)  (And my understanding would be that there doesn't need to be any expiration anyway if the invalidation logic is correct?)  But lets settle with an 8 hour expiration time as done in c6605fe350000eaa3c0de35d05f73af25453405d.